ncsa/standalone-smm-analytics
# 2 disable host ip from docker compose command

This PR does not fix anything... just put an nginx in front of the minio so we can access the admin console

TO Test:

- set `export HOST_IP=minio` in the shell script
- start up the whole stack
- go to history page it should work (but download doesn't work)
- go to localhost:9001 and login with the `aws_access_key` and `access_secret`
